makedebian.pl

```diff
@@ -331,28 +323,24 @@
 	fi
 fi
 rm -f /var/lock/subsys/$baseproduct
-which systemctl >/dev/null 2>&1 && systemctl daemon-reload
```

Comment:
Do we no longer need this?

Reply:
No, as `systemd` must and will be restarted right after `webmin.service` file changes (modified, deleted or added).

Reply:
Thanks!